core/domain/prod_validation_jobs_one_off.py

```diff
@@ -907,8 +907,8 @@ def _get_common_properties_of_external_model_which_should_not_match(
             unused_item: ndb.Model. BaseUserModel to validate.
 
         Returns:
-            list(tuple(str, str, list, str, list).
-            A list of tuple which consists of External model name,
+            list(tuple(str, str, list, str, list). A list of tuple which
+            consists of External model name,
```

Comment:
It is not good to see line break at the half stage of the line -- could you please move the part of below lines above?

Reply:
Done.